HlibSokolovskyi/HW_6/6_2_hw.py

```diff
@@ -0,0 +1,14 @@
+clear_login = ['F', 'i', 'r', 's', 't']
```

Comment:
good job

look at this implementation

user_name = input('Hello, please input your Log in:')

while user_name != 'First':
    user_name = input('Error: wrong username, please try one more time. Username:')
else:
    print('Greeting. Access granted!!!', user_name)